phoenix-core/src/main/java/org/apache/phoenix/tool/PhoenixCanaryTool.java

```diff
@@ -396,16 +324,7 @@ public int run(String[] args) throws Exception {
             Properties connProps = new Properties();
             connProps.setProperty("phoenix.schema.mapSystemTablesToNamespace", "true");
             connProps.setProperty("phoenix.schema.isNamespaceMappingEnabled", "true");
```

Comment:
Not really a part of this change but looks like we always assume that is should be true?

Reply:
Looks like they previously were trying to connect twice to get around this.  @swaroopak Please fix